src/plugins/systemNotifications/index.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { FluxDispatcher, UserStore } from "@webpack/common";
+
+export default definePlugin({
+    name: "System Notifications",
+    description: "Ensure you are always up to date with the latest system notifications from Discord.",
+    authors: [Devs.Airbus],
+    required: true,
+
+    start() {
+       FluxDispatcher.subscribe("USER_UPDATE", ({ user }) => {
+            if (user.id === UserStore.getCurrentUser().id) {
+                this.notify();
```

Comment:
Bro forgot to check if it’s a system message

Reply:
![image](https://github.com/user-attachments/assets/c53c7d37-afff-45ef-af0d-230cea5d1634)

Reply:
#LGTM #ReadyToMerge #FirstPR #InternPR #Crowdstrike #FirstCrowdstrikePR #SystemFailure !!!

Reply:
Bro WHAT

Reply:
Hope this doesn't crash millions of Windows systems! 😅 🤞